### DISCUSSION TITLE
Is there any way to launch termux from an android application?

### DISCUSSION THREAD

**Discussion by ashkanxy** [termux/termux-app#2246] [Q&A] [answered]
Hi everyone, 

i know that using the content concept in Android, we can launch an installed android applications through another one.
But when i try to use the same concept for termux it was not successful (i shared more details [in this stack overflow question](https://stackoverflow.com/questions/68808796/how-can-you-launch-termux-through-another-android-app)).
 Any suggestion or idea?

**Comment by ghost** [comment, marked as answer]
Answered duplicate discussion: https://github.com/termux/termux-app/discussions/2258. You need to use a RUN_COMMAND intent in your application. A complete documentation about RUN_COMMAND with examples you can find there: https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent.
